libra-node/Cargo.toml

```diff
@@ -39,9 +39,6 @@ storage-service = { path = "../storage/storage-service", version = "0.1.0" }
 libra-types = { path = "../types", version = "0.1.0" }
 vm-runtime = { path = "../language/vm/vm-runtime", version = "0.1.0" }
 
-[dev-dependencies]
-config-builder = { path = "../config/config-builder", version = "0.1.0" }
-
 [features]
 default = []
-fuzzing = ["libra-types/fuzzing"]
+assert-private-keys-not-cloneable = ["libra-crypto/assert-private-keys-not-cloneable"]
```

Comment:
Should we add a run with this in CI?

Reply:
Yes if you take a look at the xtask test-unit command (which is how we run unit tests now) it is run with this feature enabled in CI

Reply:
Oooh, the interesting effects of `--all-features`, of course :)